core/templates/dev/head/pages/story-viewer-page/story-viewer-page.directive.html

```diff
@@ -1,9 +1,9 @@
 <div class="oppia-story-viewer-container">
   <div ng-if="$ctrl.storyIsLoaded">
-    <story-viewer-chapters-list ng-if="$ctrl.storyPlaythroughObject.getStoryNodeCount() > 0"
-                                playthrough-object="$ctrl.storyPlaythroughObject"
-                                story-title="$ctrl.storyTitle"
-                                story-description="$ctrl.storyDescription">
-    </story-viewer-chapters-list>
+    <story-viewer-main-page ng-if="$ctrl.storyPlaythroughObject.getStoryNodeCount() > 0"
```

Comment:
Just wondering -- what is the point of having both story-viewer-page and story-viewer-main-page? Is it worth combining them into one thing?

Reply:
Yeah, deleted these files, and moved it to story-viewer-page.